Sources/Testing/ABI/EntryPoints/EntryPoint.swift

```diff
@@ -296,9 +310,11 @@ extension __CommandLineArguments_v0: Codable {
     case eventStreamVersion
     case filter
     case skip
+    case includeHiddenTests
```

Comment:
If we don't ever parse it, it doesn't need to be encoded/decoded, right?

Reply:
It has to be encoded because the ABI entry point unit tests use it, and they round-trip through the encoding representation

Reply:
That means it would be possible for external clients to set it, which we don't want.

Do we actually need the ABI entry point tests to use it, or do we just need the ability to set up an event handler to listen for warnings? Because there's two layers there: the ABI (JSON-based) event stream and the slightly higher-level all-Swift entry point, and the latter can also be set up to emit JSON if we want without requiring a JSON blob for its configuration.

Reply:
Ah yes, I can switch these tests to use the direct Swift entry point and avoid en/decoding these two new internal properties.